packaging/docker/run.sh

```diff
@@ -7,6 +7,10 @@
 # Author  : Pavlos Emm. Katsoulakis <paul@netdata.cloud>
 set -e
 
+if [ "$DO_NOT_TRACK" = 1 ]; then
```

Comment:
I think the comment on this is minor but correct. Can't we just check for existence here and  convert L64 in packaging/docker/Dockerfile to  a commented line (#ENV DO_NOT_TRACK 1)?

Reply:
Done.